libraries/networking/src/udt/Socket.cpp

```diff
@@ -147,12 +147,18 @@ Connection& Socket::findOrCreateConnection(const HifiSockAddr& sockAddr) {
 
     if (it == _connectionsHash.end()) {
         auto connection = std::unique_ptr<Connection>(new Connection(this, sockAddr, _ccFactory->create()));
+        QObject::connect(connection.get(), &Connection::connectionInnactive, this, &Socket::cleanupConnection);
         it = _connectionsHash.insert(it, std::make_pair(sockAddr, std::move(connection)));
     }
     
     return *it->second;
 }
 
+void Socket::cleanupConnection(HifiSockAddr sockAddr) {
+    qDebug() << "Cleaned up" << sockAddr;
```

Comment:
A little more context in this debug would be good for the log watcher.
Something like `"Socket::cleanupConnection called for connection to" << sockAddr;`